docs/usage/config-presets.md

```diff
@@ -134,13 +134,13 @@ To host your preset config on GitHub:
   "extends": ["github>rarkins/renovate-config"]
 ```
 
-From then on Renovate will use the Renovate config from the preset repo's default branch.
+From then on Renovate will use the Renovate config from the preset repo's base branch.
 You do not need to add it as a devDependency or add any other files to the preset repo.
 
 ## GitLab-hosted Presets
 
 It is also possible to host your preset config using just a regular GitLab repository and without needing to publish it to npmjs.
-In such cases Renovate will simply look for a renovate.json file in the default branch, (for now only the master branch is supported).
+In such cases Renovate will simply look for a `renovate.json` file in the base branch.
```

Comment:
```suggestion
In such cases Renovate will simply look for a `renovate.json` file in the default branch.
```

Reply:
@zharinov is this still true about only master branch being supported?

Reply:
We're explicitly query for default branch in `config/presets/gitlab/index.ts`, defaulting to `master` if none found. Seems like `only the master branch is supported` notice is out of date, isn't it?